src/services/users.js

```diff
@@ -35,9 +37,24 @@ module.exports = (env, jwt, database, sendEmail, mediaClient, validations) => {
     return jwt.sign({ code: random() }, { expiresIn: '24h' })
   }
 
+  let whiteListedDomains
+  try {
+    whiteListedDomains = env('WHITELISTED_DOMAINS').split(',').map(str => str.toLowerCase().trim())
+  } catch (e) {
+    console.log(`WHITELISTED_DOMAINS not set as comma delimited string of email addresses properly, error: ${e.stack}`)
```

Comment:
Please replace these with `console.error`. I've been trying to relegate `console.log` to development, so that we can tell if dev logging was accidentally left behind.